examples/acoustic/acoustic_example.py

```diff
@@ -42,7 +42,7 @@ def run(dimensions=(150, 150, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     # Smooth velocity
     initial_vp = smooth10(true_vp, dimensions)
 
-    dm = true_vp**-2 - initial_vp**-2
+    dm = 1. / true_vp * true_vp - 1. / initial_vp * initial_vp
 
```

Comment:
That is actually very bad(!), since I can now "optimise" the model and completely break it without knowing. It's also relevant to this PR though, so just a point to ponder...